src/praisonai-agents/praisonaiagents/session/title.py

```
@@ -0,0 +1,153 @@
+        ... )
+        >>> # Returns something like: "Python Code Debugging Help"
+    """
+    # Fallback title from user message if generation fails
+    fallback_title = _create_fallback_title(user_msg, max_length)
+    
+    try:
+        # Run async version in sync context
+        return asyncio.run(generate_title_async(
+            user_msg, assistant_msg, llm_model, timeout, max_length
+        ))
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)